modules/caddyhttp/fileserver/staticfiles.go

```diff
@@ -36,6 +38,29 @@ import (
 	"github.com/caddyserver/caddy/v2/modules/caddyhttp/encode"
 )
 
+type VirtualFile struct {
```

Comment:
The struct isn't used outside of the module boundaries, you can use `virtualFile` so we avoid exposing internals unnecessarily.

Reply:
You are right, thanks!

I'll take care of it as soon as we agree on the direction of this PR.